commit a284e43852380ab71eeb996389e01992d74a8dde
Author: Kees Cook <kees@kernel.org>
Date:   Wed May 1 12:37:12 2024 -0700

    hardening: Enable KCFI and some other options
    
    Add some stuff that got missed along the way:
    
    - CONFIG_UNWIND_PATCH_PAC_INTO_SCS=y so SCS vs PAC is hardware
      selectable.
    
    - CONFIG_X86_KERNEL_IBT=y while a default, just be sure.
    
    - CONFIG_CFI_CLANG=y globally.
    
    - CONFIG_PAGE_TABLE_CHECK=y for userspace mapping sanity.
    
    Reviewed-by: Nathan Chancellor <nathan@kernel.org>
    Link: https://lore.kernel.org/r/20240501193709.make.982-kees@kernel.org
    Signed-off-by: Kees Cook <keescook@chromium.org>

diff --git a/arch/arm64/configs/hardening.config b/arch/arm64/configs/hardening.config
index b0e795208998..24179722927e 100644
--- a/arch/arm64/configs/hardening.config
+++ b/arch/arm64/configs/hardening.config
@@ -5,6 +5,7 @@ CONFIG_ARM64_SW_TTBR0_PAN=y
 
 # Software Shadow Stack or PAC
 CONFIG_SHADOW_CALL_STACK=y
+CONFIG_UNWIND_PATCH_PAC_INTO_SCS=y
 
 # Pointer authentication (ARMv8.3 and later). If hardware actually supports
 # it, one can turn off CONFIG_STACKPROTECTOR_STRONG with this enabled.